.circleci/config.yml

```diff
@@ -536,6 +577,91 @@ jobs:
     steps:
       - checkout
 
+  importers:
+    parameters:
+      python_version_major:
+        type: integer
+        default: 3
+      python_version_minor:
+        type: integer
+        default: 8
+      shard:
+        type: string
+        default: "core"
+      toxenv:
+        type: string
+      codecov_flags:
+        type: string
+        default: ""
+      notify_on_failure:
+        type: boolean
+        default: false
+      xdist:
+        type: integer
+        default: 3
+      tox_args:
+        type: string
+        default: ""
+    machine: true
+    resource_class: xlarge
+    working_directory: ~/project
+    environment:
+      SHARD: << parameters.shard >>
+    steps:
+      - checkout
+      - run:
+          name: Install system deps
+          command: |
+            sudo apt-get update && sudo apt-get install -y libsndfile1 ffmpeg
+      - gcloud/install
+      - setup_gcloud
+      - run:
+          name: Decode GCP Service Account JSON
+          command: echo $GCP_SERVICE_ACCOUNT_JSON_ENCODED | base64 --decode > ${HOME}/gcp-key.json
+      - run:
+          name: Authenticate with GCP Artifact Registry
+          command: |
+            echo $GCP_SERVICE_ACCOUNT_JSON_DECODED | docker login -u _json_key --password-stdin https://<< pipeline.parameters.gcp_project >>-docker.pkg.dev
+      - start_local_testcontainer
+      - start_local_testcontainer:
+          local_base_port: "9180"
+          services_api_port: "9183"
+          fixture_service_port: "9115"
+          container_name: "wandb-local-testcontainer2"
```

Comment:
These values are hardcoded in tests/pytest_tests/system_tests/test_importers/test_wandb/conftest.py. It would be nice to pass them to the test explicitly to avoid creating an implicit dependency between this configuration file and the test. That is, it should be possible to update one without knowing about the other.

It would also help to have a comment explaining how the second testcontainer is used and why it's necessary.

Reply:
How about something like this?
https://github.com/wandb/wandb/pull/6897/commits/b42d436e6f4e4130437c1f09aeee28b1f589436e